docs/index.asciidoc

```diff
@@ -523,6 +523,13 @@ should reference only data or client nodes in Elasticsearch.
 Any special characters present in the URLs here MUST be URL escaped! This means
 `#` should be put in as `%23` for instance.
 
+When using Logstash 7.9+, this setting supports defining an environment variable
+`hosts => '${ES_HOSTS:}' with multiple {logstash-ref}/plugin-concepts.html[whitespace-delimited URIs], e.g.:
```

Comment:
Maybe we should link to the specific anchor for the subject (`{logstash-ref}/plugin-concepts.html#space-delimited-uris-in-list-params`), rather than the general plugin concepts page? The page looks like a general one that could be expanded with further entries over time.